CHANGELOG.md

```diff
@@ -12,13 +12,19 @@ To publish a new release run `scripts/release` from the project directory.
   binary.  We do recommend to also `strip` the `mdcat` binary.
 - Render SVG images in [kitty] (see [GH-114]).
 - Update to reqwest 0.10.
+- Process file list as input (see [GH-115] or [GH-54]).
+- Add --fail flag  
+  When processing a file list, exit on the first file that generates an error
+  and not process the remaining files in the list.
```

Comment:
Could you use the following instead (just copy and paste :) ):

```
- Process file list as input (see [GH-54] and [GH-115]):
    - Add `--fail` flag to exit on the first error when processing a file list;
      the default behaviour is to continue with the next file in case of error.  
```

This makes it clear that the `--fail` flag was introduced with the same change as the file list processing.

Reply:
Done,  I will be a happy camper to be able to drop this branch!